db/migrate/033_alter_key_and_index_to_bigint.rb

```diff
@@ -0,0 +1,37 @@
+class AlterKeyAndIndexToBigint < ActiveRecord::Migration[5.0]
+  def up
+    change_column :job_definitions, :id, :bigint, unsigned: false, auto_increment: true
```

Comment:
Why is `unsigned: false` needed? Isn't it default?

Reply:
you're right ;) I removed it. (rebased)